.github/workflows/genesis-verify.yml

```diff
@@ -0,0 +1,59 @@
+name: genesis-verify
+
+on:
+  pull_request:
+    branches:
+      - master
+    paths:
+      - "misc/deployments/**/genesis.json"
+
+jobs:
+  verify:
+    strategy:
+      fail-fast: false
+      matrix:
+        testnet: ["test4.gno.land", "test5.gno.land"]
```

Comment:
Right now it should only verify `test5.gno.land`

Reply:
83be3b2f0445fbb398afbf6537133325c11094b0